xchange-yobit/src/main/java/org/knowm/xchange/yobit/YoBitAdapters.java

```diff
@@ -131,7 +132,7 @@ public static String adaptCcyPairsToUrlFormat(Iterable<CurrencyPair> currencyPai
       pairs.add(adaptCcyPairToUrlFormat(currencyPair));
     }
 
-    return String.join("-", pairs);
+    return StringUtils.join(pairs, "-");
```

Comment:
String.join is only available in 1.8.
[Javadoc](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.CharSequence...-)

 Since this is 1.7 project...